src/server/uWsUtils.ts

```diff
@@ -0,0 +1,53 @@
+import { App, AppOptions, HttpRequest, HttpResponse, SSLApp, TemplatedApp } from 'uWebSockets.js';
```

Comment:
Naming of the class uWsUtils.ts -> UWsUtils.ts? :-)

Reply:
The official short name of the library is uWs, I've just reused it.

Reply:
> The official short name of the library is uWs, I've just reused it.

I know but here this is a class name and the lib doesn't have any classes that start with a minus letter with what I see in the import ;-)